### PR TITLE
And Coordinator front-end for adding and searching tenants

### PR DESCRIPTION
**Pull request by gpyofavv**
**IMPORTANT**
This pull request is NOT ready to actually be merged. Two major TODOs exist: 
1) tenant/src/Revature.Tenant.Api/ServiceBus/AddressService.cs 
- The GetAddressAsync Method had to be altered per the comment there. The real code is commented out, with dummy code used for test while we await address service's exodus into master.
2) in both add-tenant and search-tenant component, a hard coded coordinator id (training center id) exists
- I need to get this from account service, now that Jacob has made that available in master

This PR adds the Coordinator front end and tests, particularly the add-tenant and search tenant components, with accompanying service to talk to the fully operation back end.

### CODE REVIEW

File: housing/src/app/services/test-tenant-static.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    static dummyAddress: PostTenantAddress = {
+        street: '123 Sesame St',
+        city: 'Arlington',
+        state: 'TX',
+        country: 'United States',
+        zipCode: '12345'
+    };
+
+    static dummyCar: PostCar = {
+        licensePlate: 'I(╯°□°）╯︵ ┻━┻',
```
Comment (markcmoore) [inline housing/src/app/services/test-tenant-static.ts]: uuuhhhhhh.......ok.

**Comment by gpyofavv** [comment]
@escalonn , the pipeline times out: https://dev.azure.com/revaturexyz/arlington/_build/results?buildId=3658&view=logs&j=882b2ab4-82ef-5feb-f677-1931dbeee894&t=3a319992-98ff-40a8-b2d3-ce20e90d4c28

File: housing/src/app/add-tenant/add-tenant.component.ts [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  batch: Batch;
+
+  trainCen: string[] = ['fa416c6e-9650-44c9-8c6b-5aebd3f9a670'];
+  trainCenId: string;
+
+  showCarForm = false;
+  showAddressForm = false;
+
+  genderShowString = 'Choose Gender';
+  genders: string[] = ['Male', 'Female', 'Non Binary'];
```
Comment (escalonn) [inline housing/src/app/add-tenant/add-tenant.component.ts]: non-binary isn't in the backend, nor the use cases

File: tenant/src/Revature.Tenant.Api/Startup.cs [hunk 3]
```diff
@@ -37,6 +37,7 @@ public void ConfigureServices(IServiceCollection services)
         {
           builder.WithOrigins("http://localhost:4200",
                               "https://localhost:4200",
+                              "http://192.168.99.100:15080",
```
Comment (escalonn) [inline tenant/src/Revature.Tenant.Api/Startup.cs]: this doesn't belong but fine

**Pull request merged**